modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+[id="persistent-storage-csi-migration-enable_{context}"]
+= Enabling CSI automatic migration
+.Procedure
+
+* Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
++
+The following configuration example enables CSI automatic migration to both CSI drivers (EBS and Cinder):
```

Comment:
Please add a note somewhere that enablement of CSI migration will drain and restart all nodes in the cluster (in sequence) and as such may take some time.